include/bl_storage.h

```diff
@@ -10,7 +10,13 @@
 #include <string.h>
 #include <zephyr/types.h>
 #include <drivers/nrfx_common.h>
+#if defined(CONFIG_NRFX_NVMC)
 #include <nrfx_nvmc.h>
+#elif defined(CONFIG_NRFX_RRAMC)
+#include <nrfx_rramc.h>
+#else
+#error "No NRFX storage technology supported backend selected"
```

Comment:
[Here](https://github.com/nrfconnect/sdk-nrf/blob/c2e2b2e1ef91e8d7b3d17434fa04212d9fcada2d/modules/trusted-firmware-m/tfm_boards/CMakeLists.txt#L44) TF-M includes attest_hal.c.
This [includes bl_storage.h](https://github.com/nrfconnect/sdk-nrf/blob/main/modules/trusted-firmware-m/tfm_boards/common/attest_hal.c#L19).
And TF-M does not have any knowledge about Kconfig symbols, making this fail for projects including TF-M.

Reply:
The easiest solution I can think of is to also have another layer of "#if define" here to also check for TF-M, but that is maybe a bit convoluted?

Reply:
Found a fix:
Adding `#include <zephyr/autoconf.h>` to the top of bl_storage.h. 
This way TF-M will find Kconfig values, but since this file is always auto-included in zephyr, it should not mess with anything else, I think.